### ISSUE TITLE
Test on Redmine 4.0.0, 3.4.7, 3.3.9

### ISSUE THREAD

**Issue by two-pack** [two-pack/redmine_xls_export#116]
Test on Redmine 4.0.0.

**Comment by two-pack** [comment]
This plugin test has following errors on r17516 of Redmine trunk.

~~~
$ bundle exec rake redmine:plugins:test NAME=redmine_xls_export
/Users/two-pack/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/spreadsheet-1.1.8/lib/spreadsheet.rb:49: warning: already initialized constant Spreadsheet::VERSION
/Users/two-pack/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/spreadsheet-1.1.8/lib/spreadsheet.rb:49: warning: previous definition of VERSION was here
Run options: --seed 10162

# Running:

/redmine/trunk/lib/redmine/scm/adapters/subversion_adapter.rb:27: warning: already initialized constant Redmine::Scm::Adapters::SubversionAdapter::SVN_BIN
/redmine/trunk/lib/redmine/scm/adapters/subversion_adapter.rb:27: warning: previous definition of SVN_BIN was here
F

Failure:
XlsExportTest#test_details_export_is_success [/redmine/trunk/plugins/redmine_xls_export/test/integration/xls_export_test.rb:33]:
Expected response to be a <2XX: success>, but was a <422: Unprocessable Entity>


bin/rails test plugins/redmine_xls_export/test/integration/xls_export_test.rb:27

E

Error:
XlsExportTest#test_export_with_cf_which_is_selectable_multi_items:
ActiveRecord::RecordNotFound: Couldn't find User without an ID
    test/test_helper.rb:351:in `log_user'
    plugins/redmine_xls_export/test/integration/xls_export_test.rb:38:in `test_export_with_cf_which_is_selectable_multi_items'


bin/rails test plugins/redmine_xls_export/test/integration/xls_export_test.rb:37

...............................F

Failure:
QuickExportTest#test_quick_link_is_in_issues_page [/redmine/trunk/plugins/redmine_xls_export/test/integration/quick_export_test.rb:54]:
Expected false to be truthy.


bin/rails test plugins/redmine_xls_export/test/integration/quick_export_test.rb:51

...................

Finished in 131.703837s, 0.4024 runs/s, 2.6347 assertions/s.
53 runs, 347 assertions, 2 failures, 1 errors, 0 skips
Coverage report generated for MiniTest to /redmine/trunk/coverage. 2779 / 6251 LOC (44.46%) covered.
~~~

**Comment by two-pack** [comment]
I should specify **RAILS_ENV=test** .
That errors is my misstake ;(

~~~
$ RAILS_ENV=test bundle exec rake redmine:plugins:test NAME=redmine_xls_export
Run options: --seed 9315

# Running:

.....................................................

Finished in 36.001227s, 1.4722 runs/s, 9.8052 assertions/s.
53 runs, 353 assertions, 0 failures, 0 errors, 0 skips
~~~

**Comment by two-pack** [comment]
Released 4.0.0, 3.4.7, 3.3.9!
http://www.redmine.org/news/119

I finished checking.
